testing/test_pytest_html.py

```diff
@@ -383,12 +383,12 @@ def pytest_runtest_makereport(item, call):
         testdir.makepyfile('def test_pass(): pass')
         result, html = run(testdir)
         hash_key = ('test_extra_text_separated.py::'
-                    'test_pass00').encode('utf-8')
+                    'test_pass00')
         hash_generator = hashlib.md5()
-        hash_generator.update(hash_key)
+        hash_generator.update(hash_key.encode('utf-8'))
```

Comment:
I didn't get why you moved the econding from line 386 to here.

Reply:
I did the same thing in plugin.py. 
This way filename is just more clear. Without it filename would be something like: 
 b'test_name'_[hash].png

I wanted to remove useless "b" and single quotes

Reply:
Gotcha! I think your PR is nice!

Reply:
wow, thank! ^_^